test/index.js

```diff
@@ -564,6 +565,12 @@ describe('maybe', function() {
                     errorEq(Error, 'Cannot instantiate Maybe'));
     });
 
+    it('has its type definition exported', function() {
+      eq(S.MaybeType($.Number).test(S.Nothing()), true);
+      eq(S.MaybeType($.Number).test(S.Just(1)), true);
+      eq(S.MaybeType($.Number).test('abc'), false);
+    });
+
```

Comment:
GitHub only collapses comments on a diff if the commented line is changed or if one of the (two?) lines immediately above that line is changed. This is confusing because GitHub displays more context than it takes into account when deciding whether to collapse comments. It should use the same number of lines in both cases.